README.md

```diff
@@ -288,17 +288,6 @@ $ scripts/benchmark.sh
 
 Read [docs/HACKING.md](docs/HACKING.md).
 
-## Performance Demos
```

Comment:
Joran wanted these in last time I made changes to the readme. We're all cool with removing this now? (I am, but just asking.)

Reply:
Yes, it's been good so far to keep these around for posterity. I know people who've been running them or using them to get into Zig, e.g. how to work with io_uring.

Jamie actually had a comment on his blog from way back that he liked how we had preserved them—usually these kinds of things get lost along the way.

At the same time, I'm happy to prune these. They require ongoing maintenance. And it makes sense not to confuse people by lumping them in the `demos` directory.

Let's say adios!